backend/app/Recommendation/Application/UseCases/Commands/StoreRecommendationEntryCommand.php

```diff
@@ -20,8 +21,13 @@ public function __construct(
         $this->repository = app()->make(RecommendationRepositoryInterface::class);
     }
 
+    /**
+     * @throws Exception
+     */
     public function execute()
     {
+        $this->recommendation->getRecommendation();
```

Comment:
Кажется, что `getRecommendation()` должен что-то вернуть. Давай сменим нейминг

```suggestion
        $this->recommendation->perform();
```

```suggestion
        $this->recommendation->process();
```

```suggestion
        $this->recommendation->execute();
```

Reply:
executeAnswer